### DISCUSSION TITLE
[Unstable] Ruins still present in PvP levels

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14183] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

ridge, plateau, great sea and wastes pvp level still can generate ruins, which impacts load times and performance

It's very unlikely that anything happens in ruins in PvP since they're placed completely off the sides, unlike for example a rogue thalamus placed on the main path (which could be a problem like on the picture where one side is closer to the thalamus)

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/36508750-df9c-4d22-b2b1-9712d4377785)

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/pull/5499 (PR in our private repo), the fix should be coming up in the next unstable build. Closing.
